gruvbox.el

```diff
@@ -441,9 +441,9 @@ Should contain 2 %s constructs to allow for theme name and directory/prefix")
      (ivy-minibuffer-match-face-4               (:foreground gruvbox-faded_yellow))
 
      ;; ido
-     (ido-only-match                            (:foreground gruvbox-faded_green))
-     (ido-first-match                           (:foreground gruvbox-faded_green))
-     (ido-subdir                                (:foreground gruvbox-faded_red))
+     (ido-only-match                            (:foreground gruvbox-bright_green :weight 'bold))
```

Comment:
Instead of setting this explicitly to bright green, it could also inherit from `success` which is maybe a bit more idiomatic.
```suggestion
     (ido-only-match                            (:inherit 'success))
```